Tests/src/PublicAPITests/QuotaSystemTests.cpp

```diff
@@ -212,14 +212,19 @@ CSP_PUBLIC_TEST(CSPEngine, QuotaSystemTests, GetTierFeatureQuota)
 	csp::common::String UserId;
 	LogIn(UserSystem, UserId);
 
-	auto [Result] = AWAIT_PRE(QuotaSystem, GetTierFeatureQuota, RequestPredicate, TierNames::Basic, TierFeatures::OpenAI);
+	// test quota queries for basic tier
+	auto [BasicResult] = AWAIT_PRE(QuotaSystem, GetTierFeatureQuota, RequestPredicate, TierNames::Basic, TierFeatures::OpenAI);
```

Comment:
Convention for our tests is to always use `Result` for the variable name and to have these separate calls inside their own scope block.

Reply:
Ok, will do

Reply:
Addressed in latest.